File: dimik/dimik-17.c
#include <stdio.h>
#include <string.h>
int main()
{
    unsigned char repeat, i;
    char instr[1000];
    unsigned short out =0, j;
    scanf("%hhu \n", &repeat);
    for(i =0; i < repeat; i++)
    {
        scanf("%[^\n] \n", &instr);
        for(j = 0; j < strlen(instr); j++)
        {
            if (instr[j] == 'a' || instr[j] == 'e' || instr[j] == 'i' || instr[j] == 'o' || instr[j] == 'u' || instr[j] == 'A' || instr[j] == 'E' || instr[j] == 'I' || instr[j] == 'O' || instr[j] == 'U')
            {
                out = out + 1;
            }
        }
    printf("Number of vowels = %hu \n", out);
    out = 0;
    }
    return 0;
}

File: dimik/dimik-17.py
vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']
out = 0
repeat = int(input())
for x in range(repeat):
    instr = input()
    for n in range(len(instr)):
        if instr[n] in vowels:
            out = out + 1
    print('Number of vowels =', out)
    out = 0
